Dockerfile

```
@@ -1,6 +1,9 @@
-FROM golang
+FROM php:8.2.0RC4-zts-bullseye AS php-base
+
+COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
+
+RUN rm -Rf /usr/local/include/php/ /usr/local/lib/libphp.* /usr/local/lib/php/ /usr/local/php/ /usr/local/bin/*
```

```suggestion
RUN rm -Rf /usr/local/include/php/ /usr/local/lib/libphp.* /usr/local/lib/php/ /usr/local/php/ /usr/local/bin/php*
```

This is safer, other unrelated binaries may be in this directory.